test/widgets/subscription_list_test.dart

```diff
@@ -150,6 +150,32 @@ void main() {
       ]);
       check(listedStreamIds(tester)).deepEquals([2, 1, 3, 4, 6, 5]);
     });
+
+    testWidgets('channels with emoji in name are listed above non-emoji names', (tester) async {
```

Comment:
Ah I missed this last time: the test description should make it clear that it's specifically about names _starting_ with (prefixed with) emojis. Same with the other test.